src/test/java/com/bootexample4/RoostTest/credentialsGetTest.java

```diff
@@ -0,0 +1,173 @@
+// Test generated by RoostGPT for test test-postman using AI Type Open AI and AI Model gpt-4-1106-preview
+
+
+// Test generated for /credentials_get for http method type GET in rest-assured framework
+
+
+// RoostTestHash=66f52a572f
+
+
+package com.bootexample4.RoostTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import static io.restassured.RestAssured.given;
+import static org.junit.Assert.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+import org.hamcrest.MatcherAssert;
+import static org.hamcrest.Matchers.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class credentialsGetTest {
+  
+    @Test  
+    public void credentials_get_Test() {  
+        RestAssured.baseURI = System.getenv("BASE_URL");  
+  
+        // Read CSV file  
+        try (BufferedReader reader = new BufferedReader(new FileReader("src/test/java/com/bootexample4/RoostTest/credentialsGetTest.csv"))) {  
+            String headerLine = reader.readLine();  
+            String[] headers = headerLine.split(",");  
+  
+            String line;  
+            while ((line = reader.readLine()) != null) {  
+                String[] data = line.split(",");  
+  
+                // Create a map of header to data  
+                Map<String, String> map = new HashMap<>();  
+                for (int i = 0; i < headers.length; i++) {  
+                    map.put(headers[i], data[i]);  
+                }  
+                
+  
+                Response response = given()
+				.pathParam("size", map.get("size") != null ? map.get("size") : "")
+				.pathParam("offset", map.get("offset") != null ? map.get("offset") : "")
```

Comment:
it should be .queryParam("size", map.get("size") != null ? map.get("size") : "")
            .queryParam("offset", map.get("offset") != null ? map.get("offset") : "")